manuscript/01.introduction.md

```diff
@@ -1,14 +1,14 @@
 ## Introduction
 
 <!-- motivation; heterogeneity is an obstacle for chemotherapy  -->
-Chemotherapy and targeted therapies, drugs that selectively eliminate fast-proliferating or oncogene-addicted cells, are among the primary treatments of cancer. Long-term therapeutic efficacy varies significantly due in part to widespread heterogeneity in intratumor response, however [@pmid:16129367; @PMID:20619739]. Cell variability in drug response can originate from cell-intrinsic factors, such as genomic alterations and epigenetic mechanisms like changes in a chromatin state [@pmid:20371346], or cell-extrinsic factors such as spatial variability in the surrounding vasculature and environmental stressors [@doi:10.1038/nrg.2016.13; @pmid:25131830; @pmid:29250983]. Moreover, cell plasticity, where cells adopt the characteristics of other cell types, is observed more often in cancer cells, affecting their sensitivity to therapy [@doi:10.1038/bjc.2015.146].
+Chemotherapy and targeted therapies, drugs that selectively eliminate fast-proliferating or oncogene-addicted cells, are among the primary treatments of cancer. Long-term therapeutic efficacy varies significantly due in part to widespread intratumoral heterogeneity, however [@pmid:16129367; @PMID:20619739]. Cell variability in drug response can originate from cell-intrinsic factors, such as genomic alterations and epigenetic mechanisms like changes in a chromatin state [@pmid:20371346], cell-to-cell differences due to variable protein levels [@pmid:19363473], or cell-extrinsic factors such as spatial variability in the surrounding vasculature and environmental stressors [@doi:10.1038/nrg.2016.13; @pmid:25131830; @pmid:29250983]. Moreover, cell plasticity, where cells adopt the characteristics of other cell types, is observed more often in cancer cells, affecting their sensitivity to therapy [@doi:10.1038/bjc.2015.146]. 
```

Comment:
This is another good reference for variability in protein levels.

https://pubmed.ncbi.nlm.nih.gov/17122776/

Reply:
Oh yes, I will cite it here. Thanks